Bottom: 4e3aa2069062508074ecb2616f75097baa354b55
Top:    7a684ccd77496ef7f24b4f63f388bdccb4d863a1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 02:16:52 +0100

Remove unused variables Xorig and Yorig


---

diff --git a/src/data.c b/src/data.c
index 7a8f575..3cf6231 100644
--- a/src/data.c
+++ b/src/data.c
@@ -57,8 +57,6 @@ BufferType Buffers[MAX_BUFFER];	/* my buffers */
 LibraryType Library;		/* the library */
 bool Bumped;			/* if the undo serial number has changed */
 
-LocationType Xorig, Yorig;	/* origin offset for drawing in pixmap */
-
 int addedLines;
 
 double Zoom_Multiplier = 0.01;
diff --git a/src/data.h b/src/data.h
index 6e70fd3..22eea0c 100644
--- a/src/data.h
+++ b/src/data.h
@@ -59,8 +59,6 @@ extern LibraryType Library;
 
 /*extern	DeviceInfoType	PrintingDevice[];*/
 
-extern LocationType Xorig, Yorig;
-
 extern double Zoom_Multiplier;
 
 extern char *Progname;
diff --git a/src/file.c b/src/file.c
index 8a52614..4c40162 100644
--- a/src/file.c
+++ b/src/file.c
@@ -376,9 +376,6 @@ LoadPCB (char *Filename)
       Crosshair.Y =
 	MAX (0, MIN (PCB->CursorY, (LocationType) PCB->MaxHeight));
 
-      Xorig = Crosshair.X - TO_PCB (Output.Width / 2);
-      Yorig = Crosshair.Y - TO_PCB (Output.Height / 2);
-
       /* update cursor confinement and output area (scrollbars) */
       ChangePCBSize (PCB->MaxWidth, PCB->MaxHeight);
